examples/src/main/scala/spinaldoc/examples/advanced/Slots.scala

```diff
@@ -0,0 +1,68 @@
+package spinaldoc.examples.advanced
+
+import spinal.core._
+import spinal.lib._
+import scala.language.postfixOps
+
+case class SlotsDemo(slotsCount : Int) extends Component {
+  //...
+  
+  
+  //Create the hardware for each slot
+  //Note each slot is an Area, not a Bundle
+  val slots = for(i <- 0 until slotsCount) yield new Area{
+    //Because the slot is an Area, we can define mix signal, registers, logic definitions
+    //Here are the registers for each slots
+    val valid = RegInit(False)
+    val address = Reg(UInt(8 bits))
+    val age = Reg(UInt(16 bits)) //Will count since how many cycles the slot is valid
+
+    //Here is some hardware behaviour for each slots
+    //Implement the age logic
+    when(valid){
```

Comment:
It doesn't change much about the example but should there be a check against overflow? (possibly using a Counter for the age)?

Reply:
I thought about using `+|` mostly about avoiding confusion why we are doing it, but I think it's fine either way.

Reply:
Ahhh i forgot about the `+|` XDXD
hmmm still, just better focussing on the example core than adding too much distractions :D